main/swadge2024.h

```diff
@@ -29,7 +29,22 @@
  * \c demo in the example below.
  *
  * \section swadgeMode_example Example
+ * 
+ * Adding a mode to the CMakeFile requires adding two separate lines in the idf_component_register section.
+ * 
+ * \c
+ * "modes/pong/pong.c"
+ * \endcode
  *
+ * under the SRCS section and
+ * 
+ * \c
+ * "modes/pong"
+ * \endcode
+ * 
+ * under the INCLUDES section. This 
```

Comment:
This what?

Reply:
I don't know... This may be one of those times that I accidentally hit spacebar before compiling and didn't pick up on it. I also ran a bunch of things that I didn't know what they were doing behind the scenes (the various formaters), so I don't know if this is a stray "nothing".

Reply:
It read fine to me without the "this" so I made the executive decision and removed it.